tests/transforms/test_draw_mpl.py

```diff
@@ -68,6 +68,21 @@ def test_standard_use():
     plt.close()
 
 
+@pytest.mark.parametrize(
+    "strategy, initial_strategy, n_patches", [("gradient", "device", 4), ("device", "gradient", 9)]
+)
+def test_expansion_strategy(strategy, initial_strategy, n_patches):
+    @qml.qnode(qml.device("default.qubit", wires=range(2)), expansion_strategy=initial_strategy)
```

Comment:
Great! I hadn't thought about having different expansion strategies in the device and drawing.